### PR TITLE
Fix absolute value of delta m on trust-region radius

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: src/trust-region/trust-region.jl [hunk 1]
```diff
@@ -47,7 +47,7 @@ function ratio(nlp :: AbstractNLPModel, f :: Float64, f_trial :: Float64, Δm ::
   pred < 0 || throw(TrustRegionException(@sprintf("Nonnegative predicted reduction: pred = %8.1e", pred)))
 
   ared = f_trial - f + max(1.0, absf) * 10.0 * ϵ
-  if (Δm < 1.0e+4 * ϵ) || (abs(ared) < 1.0e+4 * ϵ * absf)
+  if (abs(Δm) < 1.0e+4 * ϵ) || (abs(ared) < 1.0e+4 * ϵ * absf)
```
Comment (dpo) [inline src/trust-region/trust-region.jl]: Why is this?

Comment (abelsiqueira) [inline src/trust-region/trust-region.jl]: I thought this was trying to check for small delta m, because delta m is negative.

Comment (dpo) [inline src/trust-region/trust-region.jl]: Yes I think you're right.

**Comment by dpo** [comment]
Why are we getting a build error now?
https://travis-ci.org/JuliaSmoothOptimizers/Optimize.jl/jobs/168504808#L261

**Comment by abelsiqueira** [comment]
I think `Pkg.installed` is not throwing errors anymore, but returning `nothing`. Cheking now.

**Comment by abelsiqueira** [comment]
Confirmed.

**Comment by dpo** [comment]
In what version? I'm running 0.5.0 and it throws an error.

**Comment by abelsiqueira** [comment]
That's the strangest thing. On my versions, it also throws, but the Travis version doesn't.

**Comment by dpo** [comment]
Travis uses 0.4.7 and 0.6. At least 0.4.7 should throw and error I think. It used to.

**Comment by abelsiqueira** [comment]
Tested it. For "NLPModels" and "BenchmarkProfiles" it returned nothing. For the other packages it threw an error.

**Comment by abelsiqueira** [comment]
Okay, so I think I understand it now. Packages that are on METADATA return nothing, and packages not on METADATA throw an error. Packages that are installed without METADATA return `v"0.0.0-"`.

**Comment by dpo** [comment]
Makes sense.

**Comment by abelsiqueira** [comment]
Linux-0.6 is failing with a unresolved issue: https://github.com/JuliaLang/julia/issues/18871, but everything else works.

**Comment by dpo** [comment]
Why is it ok on osx?

**Comment by abelsiqueira** [comment]
Really no idea. Nobody picked it up yet.

**Comment by abelsiqueira** [comment]
Maybe after https://github.com/JuliaLang/julia/pull/18997, it'll work.

**Comment by dpo** [comment]
Thanks!

**Pull request merged**